Source/Blazorise/Components/Rating/Rating.razor.cs

```diff
@@ -225,5 +226,10 @@ public int SelectedValue
     /// </summary>
     [Parameter] public EventCallback<int?> HoveredValueChanged { get; set; }
 
+    /// <summary>
+    /// Expression that allows the rating item to have additional user-friendly information.
+    /// </summary>
+    [Parameter] public Func<int?, RatingItemTooltipInfo> GetItemTooltipInfo { get; set; }
```

Comment:
Maybe just GetItemTooltip?

Reply:
Is there a Rating Item that's represented by null? That's why we have nullable int? Would we have an actual tooltip render when the value is null?

Reply:
Also is it ok to use `Get` prefix for `Func` from a coding style perspective? I'm not sure we use that approach, but we do seem pretty lax about it across the code base. 
Looking at some of them, could it perhaps be `ItemTooltipCreator`?

Reply:
I don't think `Creator` makes sense. This is just a delegate. So from a delegate perspective the `Get` prefix makes the most sense. And we already have that naming in other places, TreeView, for example.